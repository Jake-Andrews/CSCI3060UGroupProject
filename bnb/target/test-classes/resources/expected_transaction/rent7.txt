00_USR1___________PS____________________________________0_0.00___00
