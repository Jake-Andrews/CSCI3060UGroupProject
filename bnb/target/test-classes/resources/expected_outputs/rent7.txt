Welcome, please login to use the program. 
To exit type exit, To start type login: 
Please enter a valid username: 
You have been logged in!

Please enter a command!
For a list of commands type 'help': ERROR: You cannot rent a unit on a post-standard account!

Please enter a command!
For a list of commands type 'help': Daily transaction file string: 00_USR1___________PS____________________________________0_0.00___00

Printing Daily Transactions:00_USR1___________PS____________________________________0_0.00___00
You have been successfully logged out!
Welcome, please login to use the program. 
To exit type exit, To start type login: Thank you for using our program!